ui/package.json

```diff
@@ -13,6 +13,10 @@
     "cy:run": "cypress run",
     "test": "start-server-and-test dev http://localhost:3000 cy:run && eslint ."
   },
+  "pre-commit": [
+    "lint",
```

Comment:
Does this pre-commit hook require that contributors separately install `eslint` or will `eslint --fix` be included with the rest of the NPM packages as part of building this repo?

Reply:
Do we have an update to the Contributor guide to accompany this?

Reply:
> Does this pre-commit hook require that contributors separately install `eslint` or will `eslint --fix` be included with the rest of the NPM packages as part of building this repo?

While setting up the repository on local, we have `make setup-ui-libs` which by default installs the DevDependencies too. So we do not have to worry about the inclusion of ES-Lint in local setup, we have only ignored DevDependencies in our Production Environment to make our image size less.

Reply:
👍